packages/statistics/index.rst

```diff
@@ -0,0 +1,849 @@
+    column, and has elaborate selection and pivotal mechanisms.
+
+Creating dataframes: reading data files or converting arrays
+............................................................
+
+.. sidebar:: **Separator**
+
+   It is a CSV file, but the separator is ";"
+ 
+**Reading from a CSV file:** Using the above CSV file that gives
+observations of brain size and weight and IQ (Willerman et al. 1991), the
+data are a mixture of numerical and categorical values::
+
+    >>> import pandas
+    >>> data = pandas.read_csv('examples/brain_size.csv', sep=';', na_values=".")
+    >>> data  # doctest: +ELLIPSIS
+        Unnamed: 0  Gender  FSIQ  VIQ  PIQ  Weight  Height  MRI_Count
+    0            1  Female   133  132  124     118    64.5     816932
+    1            2    Male   140  150  124     NaN    72.5    1001121
+    2            3    Male   139  123  150     143    73.3    1038437
+    3            4    Male   133  129  128     172    68.8     965353
+    4            5  Female   137  132  134     147    65.0     951545
+    ...
+
+.. warning:: **Missing values**
+
+   The weight of the second individual is missing in the CSV file. If we
+   don't specify the missing value (NA = not available) marker, we will
+   not be able to do statistical analysis.
+
+|
+
+**Creating from arrays**: A :class:`pandas.DataFrame` can also be seen
+as a dictionary of 1D 'series', eg arrays or lists. If we have 3 numpy
+arrays::
+
+    >>> import numpy as np
+    >>> t = np.linspace(-6, 6, 20)
+    >>> sin_t = np.sin(t)
+    >>> cos_t = np.cos(t)
+
+We can expose them as a :class:`pandas.DataFrame`::
+
+    >>> pandas.DataFrame({'t': t, 'sin': sin_t, 'cos': cos_t})  # doctest: +ELLIPSIS +NORMALIZE_WHITESPACE 
+             cos       sin         t
+    0   0.960170  0.279415 -6.000000
+    1   0.609977  0.792419 -5.368421
+    2   0.024451  0.999701 -4.736842
+    3  -0.570509  0.821291 -4.105263
+    4  -0.945363  0.326021 -3.473684
+    5  -0.955488 -0.295030 -2.842105
+    6  -0.596979 -0.802257 -2.210526
+    7  -0.008151 -0.999967 -1.578947
+    8   0.583822 -0.811882 -0.947368
+    ...
+
+|
+
+**Other inputs**: `pandas <http://pandas.pydata.org>`_ can input data from
+SQL, excel files, or other formats. See the `pandas documentation
+<http://pandas.pydata.org>`_.
+
+|
+
+Manipulating data
+..................
+
+`data` is a :class:`pandas.DataFrame`, that resembles R's dataframe::
+
+    >>> data.shape    # 40 rows and 8 columns
+    (40, 8)
+
+    >>> data.columns  # It has columns   # doctest: +SKIP
+    Index([u'Unnamed: 0', u'Gender', u'FSIQ', u'VIQ', u'PIQ', u'Weight', u'Height', u'MRI_Count'], dtype='object')
+
+    >>> print(data['Gender'])  # Columns can be addressed by name   # doctest: +ELLIPSIS
+    0     Female
+    1       Male
+    2       Male
+    3       Male
+    4     Female
+    ...
+
+    >>> # Simpler selector
+    >>> data[data['Gender'] == 'Female']['VIQ'].mean()
```

Comment:
I find
```
data[data.Gender == 'Female'].VIQ.mean()
```
to be easier to read and type. But I'm not a pandas specialist at all, so maybe there are good reasons that I don't know of to use the other notation.